src/asm_cfg.cpp

```diff
@@ -97,6 +100,9 @@ static vector<label_t> unique(const std::pair<T, T>& be) {
     return res;
 }
 
+/// Get a non-deterministic version of a control-flow graph,
+/// i.e., where instead of using if/else, both branches are taken
+/// simultaneously.
```

Comment:
... and are replaced by `Assume` instructions immediately after the branch.

Reply:
Updated.   (BTW, for explicit text suggestions github has nice tooling, click on the "+-" icon on the left end of the toolbar and insert your suggested modifications, which I can then accept via a github button on the resulting comment.)